src/sentry/similarity/backends/redis.py

```diff
@@ -59,7 +61,7 @@ def _as_search_result(self, results):
         def decode_search_result(result):
             key, scores = result
             return (
-                key,
+                force_text(key),
                 map(lambda score: score_replacements.get(score, score), map(float, scores)),
```

Comment:
Afaik these `map` calls now return generators instead of lists. Guessing that's okay?

Reply:
we import our own version of map everywhere which guarantees equivalent behavior (enforced by linters)